python/iam-mfa.py

```diff
@@ -18,6 +18,9 @@
 def evaluate_compliance(configuration_item):
     if configuration_item["resourceType"] not in APPLICABLE_RESOURCES:
         return "NOT_APPLICABLE"
+    
+    if not configuration_item["configuration"]:
```

Comment:
An explicit check on the configuration item status might be better here. See https://github.com/awslabs/aws-config-rules/blob/86c3f70bc56a7b2711319f0a6dc0b12855f5cb6f/python/ec2-exposed-group.py#L62 for an example of how to do this.